spec/requests/admin/configs_spec.rb

```diff
@@ -231,42 +231,87 @@
           expected_default_image_url = URL.local_image("social-media-cover.png")
           expect(SiteConfig.main_social_image).to eq(expected_default_image_url)
 
-          expected_image_url = "https://dummyimage.com/300x300"
+          expected_image_url = "https://dummyimage.com/300x300.png"
           post "/admin/config", params: { site_config: { main_social_image: expected_image_url },
                                           confirmation: confirmation_message }
           expect(SiteConfig.main_social_image).to eq(expected_image_url)
         end
 
+        it "updates main_social_image with a valid image" do
+          expected_image = "https://dummyimage.com/300x300"
+          post "/admin/config", params: { site_config: { main_social_image: expected_image },
+                                          confirmation: confirmation_message }
+          expect(SiteConfig.main_social_image).to eq(expected_image)
+        end
```

Comment:
I added tests like this one on line `240` to ensure that an image is validated and can be uploaded even if it does _not_ contain a specific image URL (a URL ending in .`png`, for example).

cc @rhymes and @Ridhwana 😄

Reply:
This looks good! Thanks @juliannatetreault